parsing/builtin_attributes.mli

```diff
@@ -66,7 +66,7 @@ val register_attr : attr_tracking_time -> string Location.loc -> unit
 val mark_alert_used : Parsetree.attribute -> unit
 val mark_alerts_used : Parsetree.attributes -> unit
 
-(** Properties such as [@zero_alloc] that are checked
+(** Properties such as the zero_alloc attribute that are checked
```

Comment:
What's going on with this change? Why does the syntax within a comment matter?

Reply:
I think it's being misparsed by ocamldoc, this is a doc comment.